java/code/src/com/redhat/rhn/frontend/xmlrpc/contentmgmt/ContentManagementHandler.java

```diff
@@ -150,26 +150,26 @@ public ContentProject createProject(User loggedInUser, String label, String name
      * Update Content Project
      *
      * @param loggedInUser the logged in user
-     * @param label the new label
+     * @param projectLabel the new Content Project label
      * @param props the map with the Content Project properties
      * @throws EntityNotExistsFaultException when Project does not exist
      * @throws ValidationException if validation violation occurs
      * @return the updated Content Project
      *
      * @xmlrpc.doc Update Content Project with given label
      * @xmlrpc.param #session_key()
-     * @xmlrpc.param #param_desc("string", "label", "Content Project label")
+     * @xmlrpc.param #param_desc("string", "projectLabel", "Content Project label")
      * @xmlrpc.param
      *  #struct_begin("data")
```

Comment:
This must be `props` instead to match the parameter name

Also another one in `updateEnvironment`

Reply:
Right I missed that thanks :)